src/pybind/ceph_rest_api.py

```diff
@@ -20,7 +20,7 @@
 # Globals and defaults
 #
 
-DEFAULT_ADDR = '0.0.0.0'
+DEFAULT_ADDR = '::'
```

Comment:
please note, this is not a Linux specific behavior. werkzeug does not `setsockopt(sock, IPPROTO_IPV6, IPV6_V6ONLY)`, neither does the builtin `SocketServer` set this socket option. that's why we have a dual-stack socket when the app is bound to "::".